gsa/src/web/entity/icon/__tests__/cloneicon.js

```diff
@@ -39,7 +39,7 @@ describe('Entity CloneIcon component tests', () => {
     const {render} = rendererWith({capabilities: caps});
 
     const {element} = render(
-      <CloneIcon entity={entity} onClick={clickHandler} />,
+      withEmptyMock(<CloneIcon entity={entity} onClick={clickHandler} />),
```

Comment:
I guess withEmptyMock could be integrated into the render function directly because it wouldn't make a difference for components not using graphql yet.

Reply:
Hm. I don't actually know what will happen if we just wrap all components with it. Should be fine though. I'll try it.

Reply:
I am going to look at the apollo testing docs and will come up with an idea